includes/class-wc-structured-data.php

```diff
@@ -275,38 +274,46 @@ public function generate_product_data( $product = null ) {
 				'reviewCount' => $product->get_review_count(),
 			);
 
-			// Markup most recent rating/review.
+			// Markup 5 most recent rating/review.
 			$comments = get_comments(
 				array(
-					'number'      => 1,
+					'number'      => 5,
 					'post_id'     => $product->get_id(),
 					'status'      => 'approve',
 					'post_status' => 'publish',
 					'post_type'   => 'product',
 					'parent'      => 0,
-					'meta_key'    => 'rating',
+					'meta_key'    => 'rating', // phpcs:ignore WordPress.DB.SlowDBQuery.slow_db_query_meta_key
 					'orderby'     => 'meta_value_num',
```

Comment:
comment at line 277 states that we are fetching most recent ratings, but this will actually sort by the value of rating right?

Reply:
For sure we don't need it. Just need to last reviews.